### PR TITLE
Feature/umcs 383/paylater config

### PR DESCRIPTION
**Pull request by Ryouzanpaku**
- Merge feature branch into develop
- QA and review were done on feature branch.

### CODE REVIEW

File: test/integration/ConfigTest.php [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    /**
+     * Payment types that do not support config throw exception .
+     *
+     * @dataProvider verifyPaymentTypesWithNoConfigThrowExeptionDp
+     *
+     * @test
+     *
+     * @param mixed $paymentType
+     */
+    public function verifyPaymentTypesWithNoConfigThrowExeption($paymentType)
```
Comment (sixer1182) [inline test/integration/ConfigTest.php:71]: @Ryouzanpaku would it maybe make sense to return an empty config object instead of throwing an exception?

Comment (Ryouzanpaku) [reply test/integration/ConfigTest.php:71]: I would prefere to just return the API error to keep consistent with the remaining implementations. The merchant would need to handle an exeption anyways when he is doing an API request.

Comment (sixer1182) [reply test/integration/ConfigTest.php:71]: Alright

**Review by sixer1182 — APPROVED** [review]
LGTM

**Pull request merged**